CONTRIBUTING.md

```diff
@@ -1,49 +1,151 @@
+## I want to translate a page of the guide!
 
-Please note that no attempt is made to translate content for older versions of the Guides. We focus only on the current release.
+Please note that no attempt is made to translate content for older versions of the Guides. We focus only on the latest release.
 
-If you have questions, want a suggestion of what to work on, or would like a buddy to pair with, you can join the #-lang-french channel in the [Ember Community Discord](https://discordapp.com/invite/zT3asNS).
+### 1. Assign yourself a page: 
+- Find one page you want to translate in the list of [issues](https://github.com/DazzlingFugu/ember-fr-guides-source/issues) to make sure no one is already working on it.
+- If you find the issue and it's free, assign yourself to it or leave a comment to notify people you're going to start the translation.
+- If the file you wish to translate is missing from the issues list (which can happen), open a new issue using the template "File to translate".
+- Translate the page entirely and submit it for review before starting another page.
 
-## Making Your First Translation PR
+### 2. Clone the repository / Keep your fork updated
 
-- Fork this repository (click "fork" on the repository's home page in GitHub)
+If it's your first contribution, fork this repository from GitHub interface. Then on your machine:
+```
+git clone <your fork's url>
+git remote add upstream https://github.com/dazzlingfugu/ember-fr-guides-source.git
+```
+
+As you tackle new issues, you'll want to be sure that you always start by working on the most recent code. The second command is to set an upstream. By pulling from it, you can sync up your fork's `master` with a parent repository's `master`. For this to work, you should make sure you're always committing to a branch, not `master`.
+
+Each time you want to start a new page:
+```
+git checkout master
+git pull upstream master
+git checkout -b some-branch-name
+```
+
+### 3. Translate and lint the page
 
-- Clone the forked repository with `git clone <your fork's url>` and create a branch with `git checkout -b some-branch-name`.
+Each PR should have:
+- 1 commit
+- 2~3 files changed (`[some-page].md`, `.remarkignore`, optionally `.local.dic`)
 
-- Keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`
+While you are working, keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`.
```

Comment:
`feat` does not make sense with changes only focused on documentation.

```suggestion
While you are working, keep your commit stack clean with one commit, for instance: `git commit -m "docs: translate guides/release/index.md`.
```

Reply:
To me, docs refer to the documentation of the app or the addon we're implementing. Here, translating Ember Guides pages is more like our main feature IMO.

Reply:
Hmm I tend to disagree.

Working on Ember Guides, a feature would be like a new search input, a dropdown, a sidebar etc.

If people are simply updating Ember.js documentation files, whether it is for this repository or not, whether it is translations or not, it is a documentation change.

Reply:
1. Additionally, we should tell people to link their single-commit to the corresponding issue.
2. In the commit's subject, no need to tell which file is translate, since the commit already applies to a file, that's what we will see in the history and it does not make sense to write the file path in there again.

````text
While you are working, keep your commit stack clean with one commit only, and link it to the corresponding issue you are working on.

For instance, let's say we have finished to work on the issue #66 and we are ready to commit our changes:

```
git commit --all
```

Then, for the content of our commit let's write:

```
docs: translate to French

Resolves #66
```

The last line (after 2 line breaks) links your commit with the issue it resolves.
````

Reply:
Honestly, my intention was not to be so strict about the commit message, my point was just to have 1 single clear commit. What information is the most relevant is mainly a question of taste and personal way to work IMO. I'll rephrase that a bit to make it sound like advice.